README.md

```diff
@@ -1,21 +1,21 @@
-# ClickHouse MCP Server
-[![PyPI - Version](https://img.shields.io/pypi/v/mcp-clickhouse)](https://pypi.org/project/mcp-clickhouse)
+# Hydrolix MCP Server
+[![PyPI - Version](https://img.shields.io/pypi/v/mcp-hydrolix)](https://pypi.org/project/mcp-hydrolix)
 
-An MCP server for ClickHouse.
+An MCP server for Hydrolix.
 
-<a href="https://glama.ai/mcp/servers/yvjy4csvo1"><img width="380" height="200" src="https://glama.ai/mcp/servers/yvjy4csvo1/badge" alt="mcp-clickhouse MCP server" /></a>
+<a href="https://glama.ai/mcp/servers/yvjy4csvo1"><img width="380" height="200" src="https://glama.ai/mcp/servers/yvjy4csvo1/badge" alt="mcp-hydrolix MCP server" /></a>
```

Comment:
This badge needs to be replaced / removed

Reply:
I think "removed" -- I don't know if glama is the MCP server registry we'll go with, but I do know that now's not when we're deciding that.